components/BottomSheet/examples/BottomSheetTypicalUseExample.m

```diff
@@ -70,7 +70,7 @@ - (void)viewDidLoad {
 }
 
 - (void)didTapButton:(id)sender {
-  UIViewController *viewController = [[BottomSheetExampleViewController alloc] initWithCoder:nil];
+  UIViewController *viewController = [[BottomSheetExampleViewController alloc] init];
```

Comment:
Is there an advantage to using init instead of one of the designated initializers?
I would expect initWithFrame:CGRectZero over init.